impeller/entity/geometry/stroke_path_geometry.cc

```diff
@@ -46,27 +46,45 @@ class PositionWriter {
 
 class PositionUVWriter {
  public:
-  PositionUVWriter(Point texture_origin,
-                   Size texture_coverage,
+  PositionUVWriter(const Point& texture_origin,
+                   const Size& texture_size,
                    const Matrix& effect_transform)
-      : uv_transform_(Matrix::MakeScale(1 / texture_coverage) *
-                      Matrix::MakeTranslation(-texture_origin) *
-                      effect_transform) {}
+      : texture_origin_(texture_origin),
+        texture_size_(texture_size),
+        effect_transform_(effect_transform) {}
 
-  const std::vector<TextureFillVertexShader::PerVertexData>& GetData() const {
+  const std::vector<TextureFillVertexShader::PerVertexData>& GetData() {
```

Comment:
Note that I removed the "const" qualifier here so that the code could lazily update the data_ array. The corresponding method in the regular PositionWriter is still declared "const" because it has no such lazy work to do and these 2 classes are "related" but do not inherit from each other.

I was waffling on whether both should be consistent in whether they are declared "const", for symmetry, or not...

Reply:
I think that is probably fine